project.clj

```diff
@@ -24,7 +24,7 @@
 
   :test-paths ["test/common"] ;; See `test-clj` and `test-cljs` profiles below.
 
-  :profiles {:provided {:dependencies [[org.clojure/clojure "1.7.0"]]}
+  :profiles {:provided {:dependencies [[org.clojure/clojure "1.9.0-alpha10"]]}
```

Comment:
Yep. But I'm not sure how to do that. I'm gonna read up on some leiningen documentation to see if I can find something. How did we handle this before, during the `v1.8.0` transition ?